documentation/Flight and Badge Assignment.md

```diff
@@ -0,0 +1,15 @@
+# Flight and Badge Assignment
+
+Flights and badges are assigned to participants in two ways:
+
+1. Through the API
+2. Using the Director or staff's UI
+
+There is a problem - what if the participant's user object doesn't exist before the flight or badge is assigned? There are, then, two ways that a badge can be assigned:
+
+1. The user does exist, and the badge is directly assigned to the user.
```

Comment:
So the badges will be nested underneath users (when the user already exists)? Just to make sure I'm clear, we had previously shifted away from this. Are we shifting back now?